packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { IProcessedStyleSet } from '../../Styling';
 import { Label, ILabelStyleProps, ILabelStyles } from '../../Label';
+import { IconButton, IButtonProps } from '../../Button';
```

Comment:
This unfortunately pulls a lot of things in. I think we may want to use a lowercase `<button/>` here with an icon and do minimal styling.